presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionTask.java

```diff
@@ -102,7 +104,7 @@ public NativeExecutionTask(
         this.taskResultFetcher = new HttpNativeExecutionTaskResultFetcher(
```

Comment:
not related to this PR but I just found this timeout to be weird. The timeout is on the executor. But indeed, it should be at the HttpClient level. Otherwise, we are only canceling the executor and still leave the thread running.

I feel the right way is to remove all executor-related timeout and just configure the HttpClient one, which doesn't require any code change.

@rschlussel, thoughts?

Reply:
That's right. I can help to fix